src/orders/viewOrders.js

```diff
@@ -0,0 +1,40 @@
+import axios from "axios";
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/JoeMindo/USSDTest/pullRequest?prid=8695127)
```suggestion
import axios from 'axios';
```